src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,48 @@
+import './ProductCard.css';
+import PlusMinusButton from '../PlusMinusButton/PlusMinusButton';
+import { products } from '../../data';
+import { useCart } from '../../ShoppingCartContext';
+// import { useNavigate } from 'react-router-dom';
+// import { products } from '../../data';
```

Comment:
Should we get rid of unused code?

Reply:
Yes, good shout - I forgot those were there!